Infrastructure-Scripts/Set-AppRoleAssignments/config.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Could this object go in Unit.Tests.Config.Json?

Reply:
The script expects a filepath since we're going to be using it in combination with das-employer-config. Unless making some changes to the script it will have to be its own config file